elements/lisk-p2p/src/peer_directory/peer_book.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import { P2PDiscoveredPeerInfo, P2PPeerInfo } from '../p2p_types';
+import { NewPeerConfig, NewPeers } from './new_peers';
+import { TriedPeerConfig, TriedPeers } from './tried_peers';
+
+export interface PeerBookConfig {
+	readonly newPeerConfig?: NewPeerConfig;
+	readonly triedPeerConfig?: TriedPeerConfig;
+	readonly secret: number;
+}
+
+export class PeerBook {
+	private readonly _bannedPeers: ReadonlyArray<P2PDiscoveredPeerInfo>;
```

Comment:
This is not used at this point. I'd suggest to remove it to not to confuse anyone.